app/src/main/java/com/example/timemanagementapp/ui/MainActivity.kt

```diff
@@ -20,7 +21,7 @@ import kotlin.collections.ArrayList
 
 class MainActivity : AppCompatActivity() {
 
-    val DEFAULTACTIVITYNAME = "UC2020_DefaultActivity"
+    val DEFAULT_ACTIVITY_NAME = "UC2020_DefaultActivity"
 
```

Comment:
immutable val's should have capitalized names with underscores as per kotlin best practices.

https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
I agree... and good citation on naming conventions!